### ISSUE TITLE
After applying Resize resolution, the interactive map present on the "VS Code Dev Days" page is not visible: A11y_VisualStudioCodeServices_VS_CodeDevDays_Reflow.

### ISSUE THREAD

**Issue by shindepra** [Microsoft/vscode-docs#8881]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

"Contact [CAI Accessibility Support Channel](https://teams.microsoft.com/l/channel/19:DeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%40thread.tacv2/CAI%20Accessibility%20Support%20Channel?groupId=f5eb428c-9e6e-4c16-8143-33fe3961a9ae&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) for Accessibility queries on design, development, test, usability or any processes"

### GitHub Tags
#A11ySev1; #A11yTCS; #WCAG1.4.4; #ChromiumEdge; #DesktopWeb; #Win11; #Visual Studio Code Services; #SH_VisualStudioCodeServices_Web_Sept2025; #ResizeText; #Zoom; #FTP; #Benchmark;

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/dev-days
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061
ChromiumEdge Version: 141.0.3525.0 (Official build) dev (64-bit)

### Pre-Requisites: 

1. Set display resolution to 1280*768.
2. Zoom the screen at 200%.

### Repro Steps

1. Launch the application URL: https://code.visualstudio.com/dev-days
2. Observe that after applying Resize resolution, the interactive map present on the "VS Code Dev Days" page is not visible

### Actual Result
After applying Resize resolution, the interactive map present on the "VS Code Dev Days" page is not visible.

### Expected Result
After applying Resize resolution, the interactive map present on the "VS Code Dev Days" page should be visible properly.

### User Impact
User using resize resolution will not be able to see or access the interactive map present on the "VS Code Dev Days" page 

### Attachment

https://github.com/user-attachments/assets/45168f74-b49f-42a4-833d-7b346ebc9175

**Comment by ntrogh** [comment]
@jamesmontemagno Can you look into this?

**Comment by ntrogh** [comment]
@shindepra When you zoom in on the website, we switch to the mobile view and we don't show the map. The user still has access to all the events through the list of events on the page. This behavior is a design choice and is not considered an accessibility issue.

**Comment by kupatkar99** [comment]
Closing the issue as per above comment.

#Closed;

**Comment by Yash14j** [comment]
GitHub Tags
#A11yMAS;
